### ISSUE TITLE
Custom Augmentations in PixelLib

### ISSUE THREAD

**Issue by khanfarhan10** [ayoolaolafenwa/PixelLib#38]
Not quite sure how to implement a custom augmentation pipeline in PixelLib. I have used the `albumentations` library with great success, but seeing as it is implemented with `imgaug`, can you display a demo to augment with simple transforms and complex ones?

What I understood is working from `imgaug` :
```
seq = iaa.Sequential([
    iaa.Crop(px=(1, 16), keep_size=False),
    iaa.Fliplr(0.5),
    iaa.GaussianBlur(sigma=(0, 3.0))
])
```

I used in `albumentations` :

```
transform_finale = A.Compose([
    A.CLAHE (clip_limit=4.0, tile_grid_size=(8, 8), always_apply=False, p=1),
    A.HorizontalFlip(p=0.5),
    A.RandomBrightnessContrast(p=0.2),
    
])
```

Could you also provide support for  `albumentations`? 

Thanks and any help is much appreciated.

**Comment by khanfarhan10** [comment]
I searched the code and found out that this file could be modified right : https://github.com/ayoolaolafenwa/PixelLib/blob/master/pixellib/custom_train.py#L111

But I guess it would be better @ayoolaolafenwa to pass an `augmentation_pipeline` parameter and also, it can be further extended from `imgaug` to `albumentations`! Like `augmentation_pipeline` argument could be set to a default of 


```
def train_model(self,num_epochs,path_trained_models,  layers = "all", augmentation = False,augmentation_pipeline=None):

if (augmentation_pipeline==None):
  # setting the default augmentation
  augmentation_pipeline = imgaug.augmenters.Sometimes(0.5, [
                      imgaug.augmenters.Fliplr(0.5),
                      iaa.Flipud(0.5),
                      imgaug.augmenters.GaussianBlur(sigma=(0.0, 5.0))
                  ])
# else use augmentation as is.
```


A sample demo would be :
```
seq = iaa.Sequential([
    iaa.Crop(px=(1, 16), keep_size=False),
    iaa.Fliplr(0.5),
    iaa.GaussianBlur(sigma=(0, 3.0))
])
# and then calling training
train_model(self,num_epochs,path_trained_models,  layers = "all", augmentation = False,augmentation_pipeline=seq ):

```

If you wish to implement this I could modify and create a Pull Request.

**Comment by ayoolaolafenwa** [comment]
Thank you @khanfarhan10. I am presently busy and I would reach back to you later on it.

**Comment by khanfarhan10** [comment]
Fixed in #39
